package.json

```diff
@@ -108,9 +109,11 @@
     "dev": "start-storybook -s ./public -p 5555",
     "predev": "yarn",
     "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx --ignore-pattern esm --ignore-pattern lib --ignore-pattern node_modules --ignore-pattern storybook-static --quiet",
+    "lint:graphql": "eslint . --ext .graphql --ignore-pattern esm --ignore-pattern lib --ignore-pattern node_modules --ignore-pattern storybook-static",
+    "lint:fixgraphql": "eslint . --ext .graphql --ignore-pattern esm --ignore-pattern lib --ignore-pattern node_modules --ignore-pattern storybook-static --fix",
```

Comment:
For all the other lint steps, we just use the same lint command append `--fix` to the yarn command

Reply:
Good point, it's removed.